inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/group/InlongGroupRequest.java

```diff
@@ -46,10 +46,9 @@
 public class InlongGroupRequest {
 
     @ApiModelProperty(value = "Inlong group id", required = true)
-    @Length(min = 4, max = 200)
-    @Pattern(regexp = "^(?![0-9]+$)[a-z][a-z0-9_-]{1,200}$",
-            message = "inlongGroupId must starts with a lowercase letter "
-                    + "and contains only lowercase letters, digits, `-` or `_`")
+    @Length(min = 4, max = 100, message = "inlongGroupId length must be between 4 and 100")
```

Comment:
Please add @NotBlank，Otherwise, if null, the length check will be bypassed

Reply:
Done.